app/src/main/java/ahhhlvin/c4q/nyc/vinecodingproject/IssueComment.java

```diff
@@ -1,17 +0,0 @@
-package ahhhlvin.c4q.nyc.vinecodingproject;
-
-/**
- * Created by alvin2 on 2/25/16.
- */
-public class IssueComment {
-
-    public String mBody;
```

Comment:
why did you make public? make it as private. that's the whole point of making getter and setter. if you put make it public and make getter and setter, people will make fun of you. seriously lol